app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -285,6 +286,13 @@ private static boolean isDPadKey(int keyCode) {
     @Override
     public void onEntryListTouch() {
         _isDPadPressed = false;
+
+        if (_searchView != null && !_searchView.isIconified()) {
+            InputMethodManager inputMethodManager = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
+            if (inputMethodManager != null && getCurrentFocus() != null) {
```

Comment:
I think this could use a check to see whether the keyboard is actually being shown. Otherwise, we're spamming ``hideSoftInputFromWindow`` calls pretty hard while scrolling through the entry list.